tests/.babelrc

```diff
@@ -0,0 +1,9 @@
+{
+    "presets": [["env", {"modules": false}]],
+    "plugins": ["@babel/plugin-syntax-dynamic-import"],
+    "env": {
+      "test": {
+        "plugins": ["@babel/plugin-syntax-dynamic-import"]
```

Comment:
not sure why we need a babel config file?

Reply:
Ow yup forgot to remove this one. I was loading the modules with `await import('apollo-server-express')` but this is now done by: `loadPackage`.

When using dynamic `import` we needed this `.babelrc` to load the plugin when running the e2e tests, but this is not required anymore.

Reply:
fixed